packages/replay-next/src/contexts/PointsContext.tsx

```diff
@@ -34,132 +52,299 @@ export type PointInstance = {
   type: "PointInstance";
 };
 
-export type AddPoint = (partialPoint: Partial<Point> | null, location: Location) => void;
-export type DeletePoints = (...id: PointId[]) => void;
-export type EditPoint = (id: PointId, partialPoint: Partial<Point>) => void;
+export type AddPoint = (
+  partialPoint: Partial<Pick<Point, "badge" | "condition" | "content">>,
+  partialPointBehavior: Partial<Omit<PointBehavior, "pointId">>,
+  location: Location
+) => void;
+export type DeletePoints = (...pointIds: PointId[]) => void;
+export type EditPoint = (
+  id: PointId,
+  partialPoint: Partial<Pick<Point, "badge" | "condition" | "content">>
+) => void;
+export type EditPointBehavior = (
+  id: PointId,
+  pointBehavior: Partial<Omit<PointBehavior, "pointId">>
+) => void;
 
 export type PointsContextType = {
   addPoint: AddPoint;
   deletePoints: DeletePoints;
   editPoint: EditPoint;
+  editPointBehavior: EditPointBehavior;
   isPending: boolean;
+  pointBehaviors: PointBehaviorsMap;
+  pointBehaviorsForSuspense: Map<PointId, PointBehavior>;
```

Comment:
why not use `PointBehaviorsMap` here as well?

Reply:
Ha, just an oversight.